package.json

```diff
@@ -72,11 +73,17 @@
     "bluebird": "^2.9.34",
     "clean-webpack-plugin": "^0.1.3",
     "compression": "^1.5.2",
+    "css-loader": "^0.19.0",
+    "es6-promise": "^3.0.2",
```

Comment:
I had nvm set to 0.10 when I started this, and discovered that `css-loader` requires a promise polyfill now. We have fun here in JavaScript-land.